conan/internal/api/config/config_installer.py

```diff
@@ -127,14 +128,12 @@ def _process_file(directory, filename, config, cache_folder, folder):
         output.info("Copying file %s to %s" % (filename, target_folder))
         _filecopy(directory, filename, target_folder)
 
-
 def _process_folder(config, folder, cache_folder, ignore=None):
     if not os.path.isdir(folder):
         raise ConanException("No such directory: '%s'" % str(folder))
     if config.source_folder:
         folder = os.path.join(folder, config.source_folder)
-    conanignore_path = os.path.join(folder, '.conanignore')
-    conanignore = _ConanIgnoreMatcher(conanignore_path, ignore)
+    conanignore = _ConanIgnoreMatcher(find_file_walk_up(folder, ".conanignore"), ignore)
```

Comment:
Now that I see it, maybe the traverse-up has to be up to the "root" cloned repo/folder? Not just the "source_folder"?

Reply:
Mmmm, I think that it makes sense. Could you have a look at that @AbrilRBS?